### ISSUE TITLE
browser.template + webcomponent.define() = infinite loop

### ISSUE THREAD

**Issue by denis-migdal** [brython-dev/brython#2236]
Hi,

```python
from browser import ajax
from browser import html
from browser import webcomponent
from browser.template import Template

# Some d4rk magic, don't touch it!
def defineCustomElements(tagname, name, parentClass):

	def read(req):
		TEMPLATE = html.TEMPLATE(req.text)

		def init(this):
			this <= TEMPLATE.clone().content
			parameters = {k[5:] : v for k, v in this.attrs.items() if k.startswith("data-")}
			print(parameters)
			Template(this).render(**parameters)

		Class = type(name, (parentClass,), {
			'__init__': init
		})
		webcomponent.define(tagname, Class)

	x = ajax.get("./components/"+ name +"/index.html", oncomplete=read)

	
# define your custom elements here :
defineCustomElements('tp-example', 'TPExample', html.DIV)
```

```html
<div is="tp-example" data-mytext="C"></div>
```
Is giving the proper result, while
```html
<tp-example data-mytext="B"></tp-example>
```
is giving an infinite loop:
```
 texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = B texte = {mytext}
```

```
eval() error
 // Javascript code generated from ast
var $B = __BRYTHON__,
    _b_ = $B.builtins,
    locals = locals_exec,
    globals = globals_exec,
    locals_exec = locals
frame.__file__ = '<string>'
locals.__name__ = 'exec'
locals.__doc__ = _b_.None
locals.__annotations__ = locals.__annotations__ || $B.empty_dict()
$B.set_lineno(frame, 1)

var _frames = $B.frames_stack.slice()
var stack_length = $B.frames_stack.length
try{
  $B.set_lineno(frame, 1);
  $B.$call1($B.resolve_global('__write__', _frames), [0, 0, 42])('<TP-EXAMPLE data-mytext="B" >')
  $B.set_lineno(frame, 2);
  $B.$call1($B.resolve_global('__write__', _frames), [0, 0, 5981])('\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n')
  $B.set_lineno(frame, 3);
  $B.$call1($B.resolve_global('__write__', _frames), [0, 0, 34])('texte = ' + _b_.str.$factory($B.resolve_global('mytext', _frames)))
  $B.set_lineno(frame, 4);
  $B.$call1($B.resolve_global('__write__', _frames), [0, 0, 26])('</TP-EXAMPLE>')
}catch(err){
  $B.set_exc(err, frame)
  if((! err.$in_trace_func) && frame.$f_trace !== _b_.None){
    frame.$f_trace = $B.trace_exception()
  }
  throw err
}
[brython.js:6413:9](http://localhost:3000/Brython-3.11.2/brython.js)
-- python source
 __write__('<TP-EXAMPLE data-mytext="B" >')
__write__("\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n\ntexte = B\n")
__write__(f"""texte = {mytext}""")
__write__('</TP-EXAMPLE>')
[brython.js:6414:9](http://localhost:3000/Brython-3.11.2/brython.js)
Javascript error
 Error: undefined [brython.js:7766:9](http://localhost:3000/Brython-3.11.2/brython.js)
frames 
Array(800) [ (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], … ]
[brython.js:7768:9](http://localhost:3000/Brython-3.11.2/brython.js)
Javascript error
 Error: undefined [brython.js:7766:9](http://localhost:3000/Brython-3.11.2/brython.js)
frames 
Array(800) [ (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], … ]
[brython.js:7768:9](http://localhost:3000/Brython-3.11.2/brython.js)
template 259, exc <Javascript object: [object Error]>
[brython.js:14392:56](http://localhost:3000/Brython-3.11.2/brython.js)
Javascript error
 Error: undefined [brython.js:7766:9](http://localhost:3000/Brython-3.11.2/brython.js)
frames 
Array(800) [ (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], … ]
[brython.js:7768:9](http://localhost:3000/Brython-3.11.2/brython.js)
Javascript error
 Error: undefined [brython.js:7766:9](http://localhost:3000/Brython-3.11.2/brython.js)
frames 
Array(799) [ (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], (5) […], … ]
[brython.js:7768:9](http://localhost:3000/Brython-3.11.2/brython.js)
InternalError: too much recursion [brython.js:14392:56](http://localhost:3000/Brython-3.11.2/brython.js)
Uncaught Error: undefined[brython.js line 9592 > Function:1244:116](http://localhost:3000/Brython-3.11.2/brython.js%20line%209592%20%3E%20Function)
```

Cordially

**Comment by denis-migdal** [comment]
Note :
```html
<div is="tp-example" data-mytext="C"></div>
```

Seems to not work on Chromium-based browsers. I'd assume this could be due to the lack of the 3rd argument in :
```python
webcomponent.define(tagname, Class)
```

**Comment by denis-migdal** [comment]
I think I should close this issue as would likely be fixed by #2240.
